apps/docs/content/guides/functions/quickstart.mdx

```diff
@@ -1,126 +1,155 @@
 ---
 id: 'functions-quickstart'
-title: 'Developing Edge Functions locally'
-description: 'Get started with Edge Functions on your local machine.'
-subtitle: 'Get started with Edge Functions on your local machine.'
+title: 'Developing Edge Functions with Supabase'
+description: 'Get started with Edge Functions on the Supabase dashboard.'
+subtitle: 'Get started with Edge Functions on the Supabase dashboard.'
 tocVideo: '5OWH9c4u68M'
 ---
 
-Let's create a basic Edge Function on your local machine and then invoke it using the Supabase CLI.
+In this guide we'll cover how to create a basic Edge Function on the Supabase dashboard, and access it using the Supabase CLI.
 
-## Initialize a project
+## Deploy from Dashboard
 
-Create a new Supabase project in a folder on your local machine:
+Go to your project > Edge Functions > Deploy a new function > Via Editor
 
-```bash
-supabase init
-```
+<Image
+  alt="Deploy functions from the dashboard"
+  src={{
+    light: '/docs/img/guides/functions/dashboard/deploy-function--light.png',
+    dark: '/docs/img/guides/functions/dashboard/deploy-function--light.png',
+  }}
+  zoomable
+/>
 
-<Admonition type="tip" label="CLI not installed?">
+This will scaffold a new function for you.
 
-Check out the [CLI Docs](/docs/guides/cli) to learn how to install the Supabase CLI on your local machine.
+<Image
+  alt="Scaffold functions through the dashboard editor"
+  src={{
+    light: '/docs/img/guides/functions/dashboard/scaffold-function--light.png',
+    dark: '/docs/img/guides/functions/dashboard/scaffold-function--light.png',
+  }}
+  zoomable
+/>
 
-</Admonition>
+Modify the function as needed, name it, and click `Deploy function`
+
+That's it! Your function is now active. Use the endpoint on the function's Details page to call the edge function.
```

Comment:
We should suggest using Test option in the Dashboard to test the deployed function. Maybe a screenshot of Test tool is good.